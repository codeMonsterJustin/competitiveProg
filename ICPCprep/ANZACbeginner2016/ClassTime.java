import java.util.*;

public class ClassTime{
    public static void main(String[] args){

        Scanner input = new Scanner(System.in);

        int rows = input.nextInt();
        input.nextLine();

        String[] nameArray = new String[rows];

        for(int i = 0; i < rows; i++){
            String[] name = input.nextLine().split(" ");
            nameArray[i] = name[1] + " " + name[0];
        }
        Arrays.sort(nameArray);

        for(String name : nameArray){
            System.out.printf("%s %s\n", name.split(" ")[1], name.split(" ")[0]);
        }
    }
}
