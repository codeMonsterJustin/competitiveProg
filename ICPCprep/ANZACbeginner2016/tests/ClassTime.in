100
Rickey Porter
Colleen Todd
Jo Rodgers
Shelley Guzman
Mercedes Hanson
Yvette Moore
Bill Luna
Ian Luna
Anthony Luna
Victoria Luna
Allison Luna
Jeff Luna
Debbie Luna
Maggie Banks
Owen Banks
Carmen Banks
Moses Banks
Frederick Herrera
Brooke Manning
Sheryl Reeves
Shaun Lynch
Muriel Diaz
Misty Lawrence
Alfred Griffin
Yolanda Crawford
Della Steele
Robyn Warner
Fernando Wheeler
Frances Wilson
Curtis Henry
Bryan Price
Brent Christensen
Susan Stanley
Troy Bryant
Marco Sparks
Rodolfo Ferguson
Caroline Francis
Penny Holland
Sue Wise
Phil Curtis
David Russell
Darrell Walters
Brad Brady
Terence Larson
James Welch
Sally Alvarez
Courtney Roberts
Milton Hayes
Craig Dunn
Joseph Miller
Tamara Flowers
Derrick Hunt
Sandy Lindsey
Darren Nelson
Julie Soto
Andres Cook
Pamela Hale
Jesus Olson
Leticia Cruz
Russell Peters
Kyle Drake
Eva Baldwin
Lois Coleman
Judy Sutton
Angelina Harrison
Shari Harvey
Jorge Snyder
Randy Gomez
Clay Sanchez
Francis Cummings
Doreen Carson
Mary Hart
Casey Williamson
Angel Shelton
Fannie Vasquez
Shirley Poole
Clara Nichols
Clifton Delgado
Jody Mckinney
Emilio Hamilton
Marian Parsons
Geneva West
Dianna Richards
Conrad Gray
Lola Mitchell
Eula Wong
Amy Boone
Phillip Neal
Vicki Davidson
Norma Ball
Rosalie Allen
Marvin Ramsey
Anita Lawson
Enrique Robbins
Phyllis French
Luther Stephens
Priscilla Reyes
Darlene Stevenson
Ralph Boyd
Tyrone Wagner
