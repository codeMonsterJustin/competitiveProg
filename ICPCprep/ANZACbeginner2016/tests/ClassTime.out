Rosalie Allen
Sally Alvarez
Eva Baldwin
Norma Ball
Carmen Banks
Amy Boone
Jeff Bowers
Ralph Boyd
Brad Brady
Troy Bryant
Victoria Bush
Moses Carroll
Doreen Carson
Brent Christensen
Lois Coleman
Andres Cook
Yolanda Crawford
Leticia Cruz
Francis Cummings
Phil Curtis
Vicki Davidson
Clifton Delgado
Muriel Diaz
Kyle Drake
Craig Dunn
Rodolfo Ferguson
Tamara Flowers
Caroline Francis
Phyllis French
Anthony Gill
Randy Gomez
Maggie Gonzalez
Conrad Gray
Alfred Griffin
Shelley Guzman
Pamela Hale
Allison Hall
Emilio Hamilton
Mercedes Hanson
Angelina Harrison
Mary Hart
Shari Harvey
Milton Hayes
Curtis Henry
Frederick Herrera
Penny Holland
Derrick Hunt
Debbie Joseph
Terence Larson
Misty Lawrence
Anita Lawson
Sandy Lindsey
Bill Luna
Shaun Lynch
Brooke Manning
Jody Mckinney
Joseph Miller
Lola Mitchell
Yvette Moore
Ian Mullins
Phillip Neal
Darren Nelson
Clara Nichols
Jesus Olson
Marian Parsons
Russell Peters
Shirley Poole
Rickey Porter
Bryan Price
Marvin Ramsey
Sheryl Reeves
Priscilla Reyes
Dianna Richards
Enrique Robbins
Courtney Roberts
Jo Rodgers
David Russell
Clay Sanchez
Angel Shelton
Jorge Snyder
Julie Soto
Marco Sparks
Susan Stanley
Della Steele
Luther Stephens
Darlene Stevenson
Judy Sutton
Owen Swanson
Colleen Todd
Fannie Vasquez
Tyrone Wagner
Darrell Walters
Robyn Warner
James Welch
Geneva West
Fernando Wheeler
Casey Williamson
Frances Wilson
Sue Wise
Eula Wong
49
